crates/next-core/js/src/entry/app/index.d.ts

```diff
@@ -5,3 +5,4 @@ export = Anything;
 
 export const __turbopack_module_id__: string | number;
```

Comment:
We don't need this file anymore since we don't have "." imports anymore?

Reply:
Indeed. Instead, we could declare our "PAGE"/"INNER_ASSETS" modules (probably would need to rename them to "@vercel/inner/page" or "@turbopack/inner/page").